vochain/app.go

```diff
@@ -334,10 +339,10 @@ func (app *BaseApplication) GetTx(height uint32, txIndex int32) (*models.SignedT
 func (app *BaseApplication) getTxTendermint(height uint32, txIndex int32) (*models.SignedTx, error) {
 	block := app.GetBlockByHeight(int64(height))
 	if block == nil {
-		return nil, fmt.Errorf("unable to get block by height: %d", height)
```

Comment:
really? why not expose these "internal" errors (AFAIU) to the consumer and let it decide whether to treat that as "tx not found"?

Reply:
The query is for a transaction identified by a height and an index (on this height). If the height does not exist, IMO is consistent to return "the transaction you were looking for does not exist".